matminer/utils/data.py

```diff
@@ -348,6 +348,9 @@ def __init__(self):
     def get_elemental_property(self, elem, property_name):
         return self.all_element_data[str(elem)][property_name]
 
+    def get_elemental_embedding(self, elem):
+        return list(self.all_element_data[str(elem)].values())
+
```

Comment:
added this for ease as other solutions using the existing functions were messy inside the featuriser